docker/packager/deb/Dockerfile

```diff
@@ -54,6 +54,8 @@ RUN apt-get --allow-unauthenticated update -y \
             libboost-system-dev \
             libboost-filesystem-dev \
             libboost-thread-dev \
+            libboost-iostreams-dev \
+            libboost-regex-dev \
```

Comment:
>ClickHouse build check — 20/21 builds are OK

Failed due to missing libraries:

>2020-06-10 21:24:10   Could not find the following Boost libraries:
>2020-06-10 21:24:10 
>2020-06-10 21:24:10           boost_iostreams
>2020-06-10 21:24:10           boost_regex

But before this build will pass I guess docker image should be updated?

Reply:
Yes. Docker images are not tested in CI.
We can merge blindly or ask @alesapin to update beforehand.

Reply:
>We can merge blindly or ask @alesapin to update beforehand.

Should work, but I prefer to check beforehand, @alesapin can you please update docker images?

Reply:
done